tests/steps/clickhouse.py

```diff
@@ -86,3 +92,71 @@ def step_put_config(context, path, node):
         ["bash", "-c", "supervisorctl restart clickhouse-server"], user="root"
     )
     assert_that(result.exit_code, equal_to(0))
+
+
+@then("save uuid table {table} in context on {node:w}")
+def step_save_table_uuid(context, table, node):
+    query = f"SELECT uuid FROM system.tables WHERE name='{table}'"
+    ret_code, response = get_response(context, node, query)
+    assert 200 == ret_code
+
+    if not hasattr(context, "uuid_to_table"):
+        context.uuid_to_table = {}
+    context.uuid_to_table[table] = response
+
+
+def _get_uuid_dirs_with_same_prefix_name(context, table_uuid, disk, node):
```

Comment:
And this method looks too specific. Can we just check the dir with such uuid instead of getting the full list with common prefix?  Like `check_table_uuid_dir(context, uuid, disk, node)`